apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -16,12 +16,43 @@ describe('When: Use the search feature', () => {
     expect(items.length).toBeGreaterThan(1);
   });
 
-  xit('Then: I should see search results as I am typing', async () => {
+  it('Then: I should see search results as I am typing', async () => {
     await browser.get('/');
     await browser.wait(
       ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
     );
 
     // TODO: Implement this test!
+    const input = await $('input[type="search"]');
+    await input.sendKeys('javas');
+    await browser.sleep(600);
+
+    const items = await $$('[data-testing="book-item"]');
+    expect(items.length).toBeGreaterThan(1);
   });
 });
+
+describe('When: Using the snackbar', () => {
+  it('Then: I should be able to undo adding a book to the Reading List', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    const input = await $('input[type="search"]');
+    const btnUndo = await $('[data-testing="btn-undo-snackbar"]');
+    await input.sendKeys('javascript');
+    await browser.sleep(600);
+
+    const bookItemList = await $$('[data-testing="btn-add-book"]:enabled');
+    if(bookItemList) {
```

Comment:
if the `bookItemList` doesn't exist then this test will still pass. Is that what you meant to do?